### PR TITLE
Fixed build issues in SUSE and added sample rules.

### PR DESCRIPTION
**Pull request by GautamSatish**
Hello,
These are changes that I had discussed on the mailing list to get SUSE build working.

1) Commented out CentOS and Scientific OS related operations from derivatives, guide and validate of the Makefile.
2) Updated guide.xslt file for including the correct groups.
3) Updated the idents to TBD in input/xccdf/system/permissions/files.xml as suggested by @redhatrises .
4) Added OVAL check service_sshd_disabled.xml as some of the common OVAL checks inherited from the shared directory had an extended definition dependent on this.
5) Added "`<platform>multi_platform_sle</platform>`" to the appropriate OVAL checks in shared/oval.
6) Added "`# platform = multi_platform_sle`" to the appropriate fix scripts in /shared/templates/static/bash.

This still does not address the SUSE CPE issue I have discussed in #1469 . I will raise a separate PR for that in openscap.

Thank you!

### CODE REVIEW

**Comment by redhatrises** [comment]
@GautamSatish please remove CentOS and Scientific Linux from the SUSE Makefiles.

File: SUSE/11/Makefile [hunk 1]
```diff
@@ -127,10 +130,11 @@ validate-xml:
...
 	oscap ds sds-validate $(OUT)/$(ID)-$(PROD)-ds.xml
-	oscap xccdf validate-xml $(OUT)/$(ID)-centos6-xccdf.xml
-	oscap ds sds-validate $(OUT)/$(ID)-centos6-ds.xml
-	oscap xccdf validate-xml $(OUT)/$(ID)-sl6-xccdf.xml
-	oscap ds sds-validate $(OUT)/$(ID)-sl6-ds.xml
+#	Removing CentOS and Scientific Linux
+#	oscap xccdf validate-xml $(OUT)/$(ID)-centos6-xccdf.xml
+#	oscap ds sds-validate $(OUT)/$(ID)-centos6-ds.xml
+#	oscap xccdf validate-xml $(OUT)/$(ID)-sl6-xccdf.xml
+#	oscap ds sds-validate $(OUT)/$(ID)-sl6-ds.xml
```
Comment (mpreisler) [inline SUSE/11/Makefile]: Don't comment it, just remove it. It will never be uncommented for SUSE.

**Comment by GautamSatish** [comment]
@redhatrises and @mpreisler , updated the MakeFile.

**Comment by redhatrises** [comment]
Ack.

**Pull request merged**